bundle/manifests/grafana-operator-controller-manager-metrics-service_v1_service.yaml

```diff
@@ -0,0 +1,16 @@
+apiVersion: v1
+kind: Service
+metadata:
+  creationTimestamp: null
+  labels:
+    control-plane: controller-manager
+  name: grafana-operator-controller-manager-metrics-service
```

Comment:
I guess this solves: https://github.com/grafana-operator/grafana-operator/issues/449 but shouldn't this be under config/ somewhere?
My understanding of bundle/manifests that it's for operator-sdk together with OLM. Is this correct?

Reply:
Yes, this is what we need to build the bundle and index image for OLM, and also what we need to submit to Operatorhub.

Reply:
So should we create a separate file somwhere in `config/` as well?

Reply:
I need to retest this on OpenShift 4.7

Reply:
@NissesSenap a metrics endpoint and service is created by default by the operator-sdk/kustomize, no need to add any code to main.go. However, by default it is protected behind kube-rbac-proxy. I've added instructions for how to expose them directly using the kustomize method.

Reply:
Ahh then I get it. For my use-case it should be find to have behind the kube-rbac-proxy.
But I think it's good that we have the document that you created to make it super clear what people can do and not.